test/extension/handlers/topologymutation/handler.go

```diff
@@ -19,70 +19,239 @@ package topologymutation
 
 import (
 	"context"
+	"fmt"
 	"strconv"
+	"strings"
 
+	"github.com/blang/semver"
+	"github.com/pkg/errors"
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
+	intstrutil "k8s.io/apimachinery/pkg/util/intstr"
 	ctrl "sigs.k8s.io/controller-runtime"
 
+	bootstrapv1 "sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1beta1"
+	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1beta1"
 	runtimehooksv1 "sigs.k8s.io/cluster-api/exp/runtime/hooks/api/v1alpha1"
 	patchvariables "sigs.k8s.io/cluster-api/internal/controllers/topology/cluster/patches/variables"
 	infrav1 "sigs.k8s.io/cluster-api/test/infrastructure/docker/api/v1beta1"
+	"sigs.k8s.io/cluster-api/util/version"
 )
 
 // NewHandler returns a new topology mutation Handler.
 func NewHandler(scheme *runtime.Scheme) *Handler {
 	return &Handler{
+		// Add the apiGroups being handled to the decoder
 		decoder: serializer.NewCodecFactory(scheme).UniversalDecoder(
 			infrav1.GroupVersion,
+			controlplanev1.GroupVersion,
+			bootstrapv1.GroupVersion,
 		),
 	}
 }
 
+var (
+	cgroupDriverCgroupfs   = "cgroupfs"
+	versionCeilingForPatch = semver.Version{Major: 1, Minor: 24}
```

Comment:
```suggestion
	cgroupDriverPatchVersionCeiling = semver.Version{Major: 1, Minor: 24}
```
nit: can we name it something like this? 

Just feels to generic otherwise as a package-level variable in a package/file with so many other patches